src/shared/ui/SearchForm/SearchForm.css

```diff
@@ -0,0 +1,33 @@
+.search {
+  justify-content: center;
+  display: flex;
+  gap: 6px;
+}
+
+.search-input {
+    background-color: #424242;
+    color: white;
+    width: 713px;
+    height: 57px;
+    border-radius: 4px;
+    opacity: 0.8;
+    border: none;
+    padding: 4px 10px;
+    font-size: 20px;
+    font-weight: 400;
+  }
+  
+  .search-input:focus {
+    outline: none;
+  }
+  
+  .search-button {
+    background-color: #f65261;
+    color: white;
+    border: none;
+    cursor: pointer;
+    font-weight: 500;
+    border-radius: 5px;
+    font-size: 16px;
+    width: 233px;
+  }
```

Comment:
let's avoid this warning 
<img width="216" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/b08c089a-89bb-4588-8a0a-c7b313d35334">

Reply:
![Screenshot from 2023-09-08 15-24-38](https://github.com/AnnaMartis/Epam-project/assets/50263725/f7970702-f0d6-4ae8-bbed-636a7ee4cf61)

Reply:
sorry I don't have that warning , please can you tell me what the warnings says so that I can change

Reply:
it's in github only, it asks us to add an empty line in the very bottom

Reply:
done